package.json

```diff
@@ -2,7 +2,7 @@
   "name": "giving-to-stanford",
   "private": true,
   "description": "Giving to Stanford",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Not super familiar the the release process on Giving. LMK if this looks right 🤔

Reply:
I was just checking the branch diff. S'all good.

Reply:
Awesome, thanks @sherakama !